hibike/devices/LimitSwitch/limit_switch.cpp

```diff
@@ -45,7 +45,7 @@ uint8_t device_read(uint8_t param, uint8_t* data, size_t len) {
   if (MAX_PAYLOAD_SIZE - len < sizeof(uint8_t) || param >= NUM_SWITCHES || param<0) {
     return 0;
   }
-  data[0] = digitalRead(pins[param]) == LOW;
+  data[0] = digitalRead(pins[param]);
```

Comment:
Digital Read returns HIGH or LOW, which is enumed to 0 and 1. 

That's not a boolean.  Please compare so that this returns a boolean, or change the spec so the paramter expected is an int.